### PR TITLE
Ian new page

### PR DESCRIPTION
**Pull request by ianras**
Witness me!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
     <h1>Home Page Directory</h1>
     <div>
       <ul>
-        <li><a href="gale-page.html">Gale's Page</a></li>
```
Comment (hayatae) [inline index.html:14]: Could you add this line back into the index page as well? Should be able to just copy/paste it back into the file and commit. ❤️

File: ian-page-two.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<html>
+
+<head>
+  <title>Ian's Dumpster Fire of a Page; Part Deux</title>
+  <link rel="stylesheet" href="main.css" type="text/css" />
+</head>
+
+<body>
+  <div class="centered-content">
+    <h1>Metal Gear Plot</h1>
```
Comment (hayatae) [inline ian-page-two.html:53]: It doesn't make any sense, but it seems pretty solid.

File: ian-page.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <div>
+      <ol>
+        <li>Give you up</li>
+        <li>Let you down</li>
+        <li>Run around and desert you</li>
+        <li>Make you cry</li>
+        <li>Say goodbye</li>
+        <li>Tell a lie and hurt you</li>
+      </ol>
+    </div>
```
Comment (hayatae) [inline ian-page.html]: Now if only the YouTube video was embedded here...

**Review by hayatae — CHANGES_REQUESTED** [review]

File: ian-page.html (hunk 3 above)
Comment (ianras) [reply ian-page.html]: something about youtube doesn't allow embedded videos... something with youtube itself unless the video permits. unless there is some code that proves me wrong

Comment (hayatae) [reply ian-page.html]: Yeah, it was the specific video you were trying to embed. Must have been disallowed by the uploader. This one embeds just fine though:
```
<iframe width="560" height="315" src="https://www.youtube.com/embed/dQw4w9WgXcQ?autoplay=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Review by hayatae — APPROVED** [review]

**Comment by hayatae** [comment]
Hot trash. I hate it. Love it. Merging it.

**Pull request merged**